app/workers/nta_csv_worker.rb

```diff
@@ -40,7 +40,7 @@ def perform(nta_id)
     rescue => error
       logger.error "NtaCsvWorker: #{error.message}"
       logger.error error.backtrace
-      upload.update(last_error: error.message, last_error_at: Time.now)
+      upload.update(last_error: error.message[0..254], last_error_at: Time.now)
```

Comment:
My sense is that the crucial info about what the problem is is almost always in the first 255, so I wouldn't bother changing it, but leave it up to you.